app.js

```diff
@@ -20,16 +20,22 @@ import {
   PARLIAMENT_FLOW_STATUSES
 } from './config';
 
-import { syncFlowsByStatus, syncSubmittedFlows } from './lib/sync';
+import { syncFlowsByStatus, syncIncomingFlows, syncSubmittedFlows } from './lib/sync';
 import { JobManager, cleanupOngoingJobs, createJob, getJob } from "./lib/jobs";
 
-/** Schedule report generation cron job */
+/** Schedule VP flows sync cron job */
 const cronPattern = process.env.POLLING_CRON_PATTERN || '0 0 7 * * *';
-const cronJob = new CronJob(
+new CronJob(
 	cronPattern,
 	syncSubmittedFlows, // onTick
 	null, // onComplete
-	true, // start
+	false, // start
+);
+new CronJob(
+	cronPattern,
+	syncIncomingFlows, // onTick
+	null, // onComplete
+	false, // start
```

Comment:
Since these are separate crons now, I'm adding support for 2 separate cron patterns (optional)